test/html.test.js

```diff
@@ -1602,6 +1607,47 @@ QUnit.test('gopsSafeToAlignWith returns correct list', function() {
     'empty array when no gops in buffer come after currentTime');
 });
 
+QUnit.test('currentGOPStart returns time of most recent GOP', function(assert) {
+  const pts = (time) => Math.ceil(time * 90000);
```

Comment:
To avoid confusion, let's not use the ceiling of the pts values? It makes the test easier to read if exactly what was put in the buffer is what we expect our result to be

Reply:
I take the `ceil` in [the function](https://github.com/videojs/videojs-contrib-media-sources/pull/178/files#diff-7623c1ea12c03a2f94f4e9ae935330f6R104) because of possible rounding errors (and also because that's what `gopsSafeToAlignWith` does it).

Reply:
Oh, sorry I meant the the ceil used in `pts` on line 1611

Reply:
Ok wait, I think I was confusing myself, thinking somehow this would important in comparing the numbers. It's not. After thinking about it, I'm pretty sure I put this `ceil`'d here because PTS numbers are actually `int`s, but because of the way JS stores numbers, 2.2*90000=198000.00000000003.

However, I think you've got a good point that dropping the `ceil` would be worth using test data that wouldn't really happen in real life.